rm-datasource/src/main/java/io/seata/rm/datasource/DataSourceManager.java

```diff
@@ -180,7 +185,13 @@ public BranchStatus branchRollback(BranchType branchType, String xid, long branc
             UndoLogManagerFactory.getUndoLogManager(dataSourceProxy.getDbType()).undo(dataSourceProxy, xid, branchId);
         } catch (TransactionException te) {
             if (LOGGER.isInfoEnabled()) {
-                LOGGER.info("branchRollback failed reason [{}]", te.getMessage());
+                int rate = CONFIG.getInt(ConfigurationKeys.TRANSACTION_UNDO_LOG_EXCEPTION_RATE, 100);
```

Comment:
Can you provide a public logUtil so that you can control similar log printing overall, not just for undolog log output.